### PR TITLE
fix region to eu-west-1 for nightly

### PR DESCRIPTION
**Pull request by gatsbyz**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -48,7 +48,8 @@ jobs:
     name: Build Devnet
     uses: ./.github/workflows/deploy.nightly.devnet.yml
     secrets:
-      AWS_REGION: ${{ secrets.AWS_REGION }}
```
Comment (gatsbyz) [inline .github/workflows/nightly.yml]: seems like the `secrets.AWS_REGION` is set as `us-west-2` for the whole repo, so i can't change the actual secret value. wondering if i should create a temporary `AWS_REGION_NIGHTLY` for nightly builds

Comment (Stefan-Ethernal) [reply .github/workflows/nightly.yml]: @vcastellm do you have permission to administer secrets and if yes, is it safe to change it now to `eu-west-1`?

**Review by Stefan-Ethernal — COMMENTED** [review]
@vcastellm @gatsbyz there are other GH workflows that rely on `AWS_REGION` env variable, so should we change it there as well?

**Review by Stefan-Ethernal — APPROVED** [review]
Other from comments LGTM.

**Review by vcastellm — APPROVED** [review]

**Pull request merged**